README.md

```diff
@@ -27,49 +27,56 @@ cd onionperf
   + **TGen** (Shadow >= v1.11.1): cmake, glib2, igraph
   + **OnionPerf**: python
 
-The easiest way to satisfy all system dependencies is to use a package manager.
+The easiest way to satisfy all system dependencies is to use a package manager. TGen is not currently packaged and needs to be built from source.
 
 ```
 #Fedora/RedHat:
-sudo yum install gcc cmake make glib2 glib2-devel igraph igraph-devel libevent libevent-devel openssl openssl-devel python
+sudo dnf install cmake make glib2 glib2-devel igraph igraph-devel python-devel
 # Ubuntu/Debian:
-sudo apt-get install gcc cmake make libglib2.0 libglib2.0-dev libigraph0 libigraph0-dev libevent libevent-dev openssl openssl-dev python
+sudo apt install cmake make build-essential gcc libigraph0-dev libglib2.0-dev python-dev
 ```
 
-**Note**: in newer distributions, `libevent` may be called `libevent-2.0` and `openssl-dev` may be called `libssl-dev`.
-
 ### Install Python modules
 
-  + **OnionPerf** python modules: stem (>= v1.4.0), lxml, networkx, numpy, matplotlib.
+  + **OnionPerf** python modules: stem (>= v1.7.0), lxml, networkx, numpy, matplotlib.
 
 #### Option 1: Package Manager
 
 The easiest way to satisfy all system dependencies is to use a package manager.
 
 ```
 # Fedora/RedHat:
-sudo yum install python-stem python-lxml python-networkx python-matplotlib numpy scipy
+sudo dnf install python-stem python-lxml python-networkx python-matplotlib numpy scipy
```

Comment:
We *only* support Debian stable (with notes about how to make it go in testing if necessary). Let's take this oppertunity to drop rpm-based and Ubuntu instructions.